lib/workos/types/user_struct.rb

```diff
@@ -6,6 +6,7 @@ module Types
     # This UserStruct acts as a typed interface for the User class
     class UserStruct < T::Struct
       const :id, String
+      const :object, String
```

Comment:
i assume the exclusion here was intentional (we use the `object` to determine which type of object to serialize from the json response, but i don't think we include it in the serialized objects, at least in the SDKs i've been working on). do we include the `object` property in other structs here in the Ruby SDK?

Reply:
otherwise the path changes look good!

Reply:
oh ok looks like we are inconsistent. here are some w/o the object property:
https://github.com/workos/workos-ruby/blob/main/lib/workos/types/directory_struct.rb
https://github.com/workos/workos-ruby/blob/main/lib/workos/types/organization_struct.rb
https://github.com/workos/workos-ruby/blob/main/lib/workos/types/event_struct.rb

Reply:
Those are missing, I think we should fix it later!

Reply:
But I think ideally all of the SDKs should return the object property as we also display it on our docs response object.

Reply:
oh i was using those examples to show that i don't think including `object` is required...

Reply:
Yeah, I'm actually not sure. I see that we display those on our docs, so I don't think there could be a case where developers expect that we return those as well within the response instead of omitting them.  

![CleanShot 2023-11-20 at 14 39 05](https://github.com/workos/workos-ruby/assets/48022589/7ee397fd-3bf6-441d-bc99-6ffd7cd290ae)

Reply:
But as you mentioned, we are not that consistent about it. Some SDKs do that, but not in all modules.

Reply:
yeah, here's how i see it:

* for json responses, like if a developer is integrating without using one of our SDKs, they need the `object` property of the json response to know the type of object
* however, when using an SDK, the developer is interacting with serialized objects, like a User or an Event. they can identify which object they are using by the class/instance type. and in that scenario, the object property is redundant

i think we should leave it out for now

Reply:
(1) is why `object` needs to be documented in the API Reference (for json/curl code snippets)

Reply:
Your point makes sense to me. Let's stop adding :object to the response structs here then - prob also be mindful of that for the other SDKs.

Reply:
sounds good, thank you!